js/app.js

```diff
@@ -5,35 +5,48 @@ const books = document.querySelector('#books-list');
 const addBtn = document.getElementById('form');
 let booksArray = [];
```

Comment:
- [ ] good job refactoring your code 👍 however according to the requirements you should use JavaScript classes instead of objects and arrays. You can use objects and arrays, but do it inside classes.

Reply:
This issue has been well addressed. Good job! :+1: :clap: